src/goto-instrument/cover_basic_blocks.h

```diff
@@ -18,7 +18,46 @@ Author: Daniel Kroening
 
 class message_handlert;
 
-class cover_basic_blockst
+class cover_blocks_baset
```

Comment:
No need to rename this. The base class should provide the language-independent (=goto) case.